liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerXmlDocument.java

```diff
@@ -152,20 +201,20 @@ private static boolean isFoundInConfigDropinsDir(String serverFilePath, File con
         if (configDropins.exists()) {
             File overrides = new File(configDropins, "overrides");
             // <server_name>/configDropins/overrides
-            if (overrides.exists() && isFoundConfigInFragmentDir(overrides)) {
+            if (overrides.exists() && isFoundAppConfigFromFragmentDir(overrides, fileName)) {
                 return true;
             }
             
             // <server_name>/configDropins/defaults
             File defaults = new File(configDropins, "defaults");
-            if (defaults.exists() && isFoundConfigInFragmentDir(defaults)) {
+            if (defaults.exists() && isFoundAppConfigFromFragmentDir(defaults, fileName)) {
                 return true;
             }
         }
         return bFound;
     }
     
-    private static boolean isFoundConfigInFragmentDir(File dir) {
+    private static boolean isFoundAppConfigFromFragmentDir(File dir, String fileName) {
         // FileFilter to get ".xml" files
```

Comment:
Are you expecting other types of files in the dropinConfigs directory which are not server configuration files?  If they are all xml files with ".xml" file extension, why you need to filter them. if they are xml files without ".xml" file extension, you should not skip them.